### DISCUSSION TITLE
Reconnect

### DISCUSSION THREAD

**Discussion by svenkoebrich** [fsaris/EspHome-AwoX-BLE-mesh-hub#49] [Q&A]
Hallo zusammen,
ich habe die Eglo Leuchten mit dem ESP32 verbunden bekommen, läuft soweit, allerdings wenn ich HomeAssistant neustarte oder die Leuchten spannungslos werden und dann wieder zugeschaltet werden bricht die Verbindung zum ESP ab.
nach neu Installation des ESP läuft alles wieder.
Gibt es eine Möglichkeit das der ESP wenn er die Geräte wieder verfügbar sind sich automatisch neu zu connecten?
 
Danke für eure Hilfe

Gruß
Sven

**Comment by fsaris** [comment]
Hi, 

The lights should automatically show up again. But maybe you ran into this issue #50? Can you check your HA logs and post the logs form ESPHome here?



ps. please try to keep the communication in English so most people can follow.
